### PR TITLE
commit for code challenge

### PR DESCRIPTION
**Pull request by EvansHG**
I used node version "6.17.1" and npm "6.10.1".

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <link rel="stylesheet" href="./css/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body>
+    <main class="main" id="app"></main>
+    <script src="./js/main.js"></script>
```
Comment (nyanofthemoon) [inline client/dist/index.html:18]: @EvansHG What is the purpose of the `./`?

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "wall",
+  "version": "1.0.0",
+  "description": "a very simple app that allows user to write things on a wall.",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open --config webpack.dev.js",
```
Comment (nyanofthemoon) [inline client/package.json:8]: @EvansHG There were no documentation on how to set-up the client module. I did the usual `npm i` and then `npm start`:

image

**Review by nyanofthemoon — CHANGES_REQUESTED** [review]
Would like to be able to build the projet to see the demo. :)

**Comment by EvansHG** [comment]
Sorry, I forgot to add .babelrc...It should work now by doing exactly just what you did (npm i and npm start  same for client and server). I hope everything will go smoothly.

step1

step2

step3

**Pull request closed without merging**